### PR TITLE
Update test_data.config

### PR DESCRIPTION
**Pull request by saulpierotti**
## PR checklist

Test data for the stitch module (#3756). Data already merged (https://github.com/nf-core/test-datasets/pull/988)

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Comment by LouisLeNezet** [comment]
Hi @saulpierotti ,

I didn't checked in the previous PR where the new file was stored. (https://github.com/nf-core/test-datasets/pull/988)
It should've been in the `delete_me` file I think.
Until we have proper datatest workflow we should stick to this folder I think.

Maybe a thought from the nf-core team would be helpfull here ?

**Comment by saulpierotti** [comment]
For reference, https://github.com/nf-core/test-datasets/pull/988 has been also modified by https://github.com/nf-core/test-datasets/pull/990 because I calculated it on chr22 instead than 21, and the cram files are vailable for chr21. On my side I don't think this is likely to change, it is a minimal file with a list of positions to test the imputation over. Since it is calculated directly from other files in the same folder I placed it there but thoughts welcome :)

In any case this should not be merged since refers to the old file

**Comment by saulpierotti** [comment]
I modified it now to point to the correct file, so it can be merged from my side if the nf-core team approves :)

**Review by LouisLeNezet — APPROVED** [review]

**Pull request merged**
